package/lib/Markdown.astro

```diff
@@ -1,11 +1,13 @@
 ---
-import type { SanitizeOptions } from 'ultrahtml'
+import type { SanitizeOptions } from 'ultrahtml/transformers/sanitize'
 import { createComponentProxy, markdown } from './utils';
+import type { MarkedExtension } from 'marked';
 
 export interface Props {
   content?: string;
   sanitize?: SanitizeOptions;
   components?: Record<string, any>;
+  markedExt?: MarkedExtension[]
```

Comment:
Slight preference for nesting this under a top-level `marked` object and using the full name `extensions`. I try to avoid using abbreviated names in public APIs where possible, because it assumes that users are already familiar with the abbreviation. It adds one more piece of context to hold in your head when using the API. Full names are more verbose, but offer comparatively less cognitive load on the user.

This is also another feature to callout in a changeset! This would be a `minor` since it adds a feature, and multiple changesets are combined into a single changelog when the PR is merged.